### PR TITLE
change test'name and algorithm

### PR DESCRIPTION
**Pull request by MarianBastiurea**
Hope to be on the good way with tests name

### CODE REVIEW

File: src/test/java/org/example/c03arrays/calculate/SumOfAllElementsOfArrayTest.java [hunk 1]
```diff
@@ -1,27 +1,31 @@
...
+        assertEquals(result, sum);
     }
+
     @Test
-    public void sumNullBecauseSameNegativeAndPositiveElements(){
-        int[] myArray={-2,-1,0,1,2};
-        int sum=0;
-        int result=SumOfAllElementsOfArray.findSum(myArray);
-        assertEquals(result,sum);
+    public void sumNullBecauseSameNegativeAndPositiveElements() {
```
Comment (andreiursudev) [inline src/test/java/org/example/c03arrays/calculate/SumOfAllElementsOfArrayTest.java]: Sum is 0, not null

File: src/test/java/org/example/c03arrays/delete/DeleteElementAtPositionInArrayTest.java [hunk 2]
```diff
@@ -6,15 +6,15 @@
...
+    public void givenPosition2ForDeleting3ShouldDisappeared() {
         int[] myArray = {1, 2, 3, 4, 5};
         int positionToDeleteAnElement = 2;
         int[] result = DeleteElementAtPositionInArray.deleteElementAtPosition(myArray, positionToDeleteAnElement);
         assertArrayEquals(result, new int[]{1, 2, 4, 5});
     }
 
     @Test
-    public void oneElements() {
+    public void givenPosition1ToBeDeletedIn1ElementArrayShouldReturnAnEmptyArray() {
```
Comment (andreiursudev) [inline src/test/java/org/example/c03arrays/delete/DeleteElementAtPositionInArrayTest.java]: givenPosition ToBeDeletedposition is 0.

File: src/test/java/org/example/c03arrays/filter/FindTwoRepeatingElementsInAnArrayTest.java [hunk 3]
```diff
@@ -6,17 +6,17 @@
 
 public class FindTwoRepeatingElementsInAnArrayTest {
     @Test
-    public void sevenElements() {
+    public void GivenMoreThan2RepeatingElementShouldReturnFirst2repeatingElements() {
```
Comment (andreiursudev) [inline src/test/java/org/example/c03arrays/filter/FindTwoRepeatingElementsInAnArrayTest.java]: the input does not contain more than 2 repeating elements

File: src/test/java/org/example/c03arrays/calculate/SumOfAllElementsOfArrayTest.java (hunk 1 above)
Comment (MarianBastiurea) [reply src/test/java/org/example/c03arrays/calculate/SumOfAllElementsOfArrayTest.java]: Done

File: src/test/java/org/example/c03arrays/delete/DeleteElementAtPositionInArrayTest.java (hunk 2 above)
Comment (MarianBastiurea) [reply src/test/java/org/example/c03arrays/delete/DeleteElementAtPositionInArrayTest.java]: Done

File: src/test/java/org/example/c03arrays/filter/FindTwoRepeatingElementsInAnArrayTest.java (hunk 3 above)
Comment (MarianBastiurea) [reply src/test/java/org/example/c03arrays/filter/FindTwoRepeatingElementsInAnArrayTest.java]: Done

File: src/test/java/org/example/c03arrays/delete/DeleteElementAtPositionInArrayTest.java [hunk 4]
```diff
@@ -14,7 +14,7 @@ public void givenPosition2ForDeleting3ShouldDisappeared() {
     }
 
     @Test
-    public void givenPosition1ToBeDeletedIn1ElementArrayShouldReturnAnEmptyArray() {
+    public void givenPosition1ToBeDeletedPositionIs0() {
```
Comment (andreiursudev) [inline src/test/java/org/example/c03arrays/delete/DeleteElementAtPositionInArrayTest.java]: Wrong test name. given position to be deleted is 0

Comment (MarianBastiurea) [reply src/test/java/org/example/c03arrays/delete/DeleteElementAtPositionInArrayTest.java]: Done

**Comment by andreiursudev** [comment]
approve

**Review by andreiursudev — APPROVED** [review]

**Pull request merged**